Project/Source/Components/UI/ComponentSlider.cpp

```diff
@@ -238,6 +270,16 @@ float ComponentSlider::GetNormalizedValue() const {
 	return normalizedValue;
 }
 
+void ComponentSlider::ModifyValue(float mulitplier) {
+#if GAME
+	currentValue = Max(Min(currentValue + mulitplier * App->time->GetDeltaTime() * sliderSensitivity, maxValue), minValue);
+#else
+	currentValue = Max(Min(currentValue + mulitplier * App->time->GetRealTimeDeltaTime() * sliderSensitivity, maxValue), minValue);
+#endif
+	if (mulitplier != 0.0f)
+		OnValueChanged();
```

Comment:
Ifs like this, separated in two lines, are prone to errors. Either move OnValueChanged to the same line or add braces.

Reply:
I forgot that single line if were kinda forbidden